cpp/CMakePresets.json

```diff
@@ -9,6 +9,19 @@
       "binaryDir": "${sourceDir}/out/${presetName}-build",
       "installDir": "${sourceDir}/out/install"
     },
+    {
+      "name": "common_conda",
+      "hidden": true,
+      "description": "OS-agnostic config",
+      "binaryDir": "${sourceDir}/out/${presetName}-build",
+      "installDir": "${sourceDir}/out/install",
+      "cacheVariables": {
+        "ARCTICDB_USING_CONDA": "ON",
+        "ARCTICDB_USING_VCPKG": "OFF",
+        "STATIC_LINK_STD_LIB": "OFF",
+        "HIDE_LINKED_SYMBOLS": "OFF"
```

Comment:
I believe these are out of sync with another PR. For example, `ARCTICDB_USING_VCPKG` is gone now.

Also, can `STATIC_LINK_STD_LIB` and `HIDE_LINKED_SYMBOLS` be driven off `ARCTICDB_USING_CONDA`? I really want to simplify the profiles at some point.

Reply:
> I believe these are out of sync with another PR.

Which PR are you thinking about? Are you thinking about https://github.com/man-group/ArcticDB/pull/352?